operationsgateway_api/src/config.py

```diff
@@ -62,6 +63,7 @@ class EchoConfig(BaseModel):
     access_key: SecretStr
     secret_key: SecretStr
     bucket_name: StrictStr
+    cache_maxsize: NonNegativeInt = 128
```

Comment:
Do we need to add this to the `config.yml.example`? Also might be worth adding a comment there as to what its used for.

Reply:
Since it takes a default I was being lazy by not including it but being more explicit can't hurt. Especially since there's some nuance in the fact that it's "enabled" by default, and to "disable" you would need to set it to 0.